src/charm.py

```diff
@@ -97,92 +100,116 @@ def _config(self) -> str:
 
         return yaml.dump(config)
 
-    def _update_layer(self) -> None:
-        """Updates the Pebble configuration layer and kratos config if changed."""
-        if not self._container.get_plan().to_dict():
-            self.unit.status = MaintenanceStatus("Applying new pebble layer")
-            self._container.push(self._config_file_path, self._config, make_dirs=True)
-            with open("src/identity.default.schema.json", encoding="utf-8") as schema_file:
-                schema = schema_file.read()
-                self._container.push(self._identity_schema_file_path, schema, make_dirs=True)
-            self._container.add_layer(self._container_name, self._pebble_layer, combine=True)
-            logger.info("Pebble plan updated with new configuration, replanning")
-            self._container.replan()
-        else:
-            # Compare changes in kratos config
-            current_config = self._container.pull(self._config_file_path).read()
-            if current_config != self._config:
-                self.unit.status = MaintenanceStatus("Updating Kratos Config")
-                self._container.push(self._config_file_path, self._config, make_dirs=True)
-                logger.info("Updated kratos config")
-                self._container.restart(self._container_name)
-
     def _get_database_relation_info(self) -> dict:
         """Get database info from relation data bag."""
         relation_id = self.database.relations[0].id
+        print(
+            f"DEBUGGING ~ file: charm.py ~ line 103 ~ self.database.relations: {self.database.relations}"
```

Comment:
Is it still relevant? Should there be a log entry instead?

Reply:
good catch, thanks